eslint.config.mjs

```diff
@@ -0,0 +1,78 @@
+import nextPlugin from '@next/eslint-plugin-next'
```

Comment:
The eslint-config-next package contains three packages:

    eslint-plugin-react
    eslint-plugin-react-hooks
    eslint-plugin-next.

Refs: https://nextjs.org/docs/app/api-reference/config/eslint#reference